go/vt/tabletserver/codex_test.go

```diff
@@ -11,14 +11,14 @@ import (
 	"testing"
 
 	"github.com/youtube/vitess/go/sqltypes"
-	"github.com/youtube/vitess/go/vt/schema"
 
 	querypb "github.com/youtube/vitess/go/vt/proto/query"
 	vtrpcpb "github.com/youtube/vitess/go/vt/proto/vtrpc"
+	"github.com/youtube/vitess/go/vt/tabletserver/engines/schema"
 )
 
 func TestCodexBuildValuesList(t *testing.T) {
-	tableInfo := createTableInfo("Table",
+	table := createTable("Table",
```

Comment:
If everybody using this method will always use &table, how about returning a *Table instead of a Table?